### ISSUE TITLE
Paradox games on Steam not time tracking properly

### ISSUE THREAD

**Issue by awbooze** [JosefNemec/PlayniteExtensions#9]
**Check for existing issue**
I did not find anything when searching for the word "paradox" or looking through the results for "steam" on GitHub.

**Describe the bug**
I play Paradox games (Stellaris and Crusader Kings III) quite a bit and launch all of my games through Playnite. For recent Paradox games (the above, but not Crusader Kings II, for instance), Paradox uses a launcher to allow you to configure mods and DLC access, as well as jump straight back into your current save (the Paradox Launcher v2, which is apparently [different than the Paradox Launcher](https://support.paradoxplaza.com/hc/en-us/articles/115003166953-Paradox-Launcher-FAQ)). When launching the games through Steam, the launcher will come up and then allow you to start the game.

Recently (although I'm not quite sure when), this broke time tracking for those games. Clicking the play button in Playnite will launch the game through Steam, but almost immediately, the play button will go back to displaying "play" and the session will not be tracked for time. I have confirmed that this is the case in safe mode (with no plug-ins) as well.

**Work Around**
If you just launch the games through their executables, time tracking works fine, although the launcher is not used. I'll be doing this for now.

**Diagnostics ID**
cf81e4b1-110c-4d58-9e54-a74538f76b25, if you need it.

**Comment by awbooze** [comment]
That's quite possible. On my system, the launcher is installed in `AppData\Local\Programs\Paradox Interactive`, but there's a bunch of settings files scattered all around the `AppData\Local` and `AppData\Roaming` folders. 

While I don't own Cities: Skylines, I would assume that it would reproduce because the help article I linked above states that it is one of the games the Paradox Launcher v2 will run with.

**Comment by awbooze** [comment]
This was an issue on my end - feel free to close this, especially if it doesn't reproduce for you.

I like to keep install games in a different folder on my `C:` drive than the rest of my programs, but Steam's Library Folder system wasn't letting me do that. I finally got around to tricking Steam into moving them about two months ago and didn't change the install directories in Playnite. I noticed this yesterday when No Man's Sky didn't initially time track either and today I finally made the connection that it couldn't have been the launcher, it was the fact that I had moved the games without changing their installation directory in Playnite.
